pkg/types/openstack/platform.go

```diff
@@ -28,6 +28,33 @@ type Platform struct {
 	// Existing Floating IP to associate with the OpenStack load balancer.
 	LbFloatingIP string `json:"lbFloatingIP"`
 
+	// APIVIP
+	// The internal virtual IP address (VIP) put in front of the
+	// Kubernetes API server for use by components inside the cluster.
+	// The DNS static pods running on the nodes resolve the api-int
+	// record to APIVIP.
+	//
+	// The value is set by the installer from the MachineCIDR range.
```

Comment:
Can the users change this value?? If yes is there validation that it belongs to the machinecidr...?

Reply:
Good catch, thanks!

The IPI users are not supposed to set this, but it is possible to do so in `install-config.yaml`. In that case there is no validation there. I'm going to add it.

Reply:
> The IPI users are not supposed to set this

will anything fail if they do??

Reply:
It didn't before but it will now.